components/brave_wallet/brave_wallet_utils.h

```diff
@@ -35,6 +37,18 @@ bool ConcatHexStrings(const std::string& hex_input1,
 bool HexValueToUint256(const std::string& hex_input, uint256_t* out);
 std::string Uint256ValueToHex(uint256_t input);
 
+// Generate mnemonic from entropy following BIP39.
+// If entropy is empty, we will generate a random 128 bits one.
+// If entropy size is not in 128-256 bits range or allocation failure, the empty
```

Comment:
Should be specified range: `128, 160, 192, 224, 256`.

Reply:
fixed in ca4d806d885cb83113b7163a9861ffdbfde28203